README.md

```diff
@@ -109,15 +133,34 @@ Your Answer:
 What are the differences between calling and referencing a function? Please provide examples of each.
 
 ```text
+when you call a function, that code runs. when you reference a function (for example if you use a function inside of another function), the program is made aware of the output of the function, but it doesn't neccesarily execute it.
+
+
+function multiply( num1, num2 ){
+  console.log( num1 * num2 );
+  return num1 * num2;
+
+(called)
+
+multiply(2,6)
+
+(referenced)
 
+var x = function {
+  var y = 2;
+  var z = y * multiply;
+  return var z;
+};   
 ```
```

Comment:
To reference a function is to write it out without using parentheses. In doing so, we return the function definition without actually executing the function itself. See the example below...

```js
function sayHello( ){
  console.log("hello");
}

// invoking
sayHello( );

// referencing
sayHello;
```

Encourage you to test that out in the console.